src/System.Text.RegularExpressions/ref/project.json

```diff
@@ -1,6 +1,7 @@
 {
   "dependencies": {
-    "System.Runtime": "4.0.0"
+    "System.Runtime": "4.0.0",
+    "System.Collections": "4.0.0"
```

Comment:
What depends on System.Collections? IDictionary is defined in System.Runtime and I didn't see anything else in the contract that looks to be in System.Collections.